iothub/device/samples/getting started/ReadD2cMessages/README.md

```diff
@@ -0,0 +1,57 @@
+# Read device-to-cloud messages
+
+This sample demonstrates how to use the Azure Event Hubs client library for .NET to read messages sent from a device by using the built-in Event Hub that exists by default for every IoT Hub instance.
```

Comment:
@schoims Was this supposed to come into previews/v2?

Reply:
this sample is in iothub/service/samples and need to be removed from device/samples/getting started

Reply:
I think it is already in the service samples. I wonder why when merging it came into the device folder too?!

Reply:
Because it is wrong in main. Will you be fixing it in main?

https://github.com/Azure/azure-iot-sdk-csharp/tree/main/iothub/device/samples/getting%20started/ReadD2cMessages

Reply:
I see.  I will fix it in main